dspace/config/dspace.cfg

```diff
@@ -780,7 +780,7 @@ event.dispatcher.default.class = org.dspace.event.BasicDispatcher
 # Add rdf here, if you are using dspace-rdf to export your repository content as RDF.
 # Add iiif here, if you are using dspace-iiif.
 # Add orcidqueue here, if the integration with ORCID is configured and wish to enable the synchronization queue functionality
-event.dispatcher.default.consumers = versioning, discovery, eperson, submissionconfig, qaeventsdelete, ldnmessage
+event.dispatcher.default.consumers = versioning, discovery, eperson, qaeventsdelete, ldnmessage
```

Comment:
@toniprieto : Should the "submissionconfig" consumer be entirely removed? (see lines 833-835) along with the `org.dspace.submit.consumer.SubmissionConfigConsumer` class?  Or is there still a use case for that SubmissionConfigConsumer?

If there is a use case for the SubmissionConfigConsumer, then we may want to enhance the comments on line 833 to explain when sites may want to enable it.

Reply:
I had the same doubt. From the tests I've done, the consumer can be useful for reloading configuration changes in item-submission.xml without restarting Tomcat, mainly for adding new collection mappings. I ended up adding a comment in dspace.cfg, although I think it's a rare use case. If you think it's better to remove it entirely, let me know.